tests/Integration/Behaviour/Features/Scenario/Product/add_virtual_product_file.feature

```diff
@@ -0,0 +1,93 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags virtual-product-file
+@reset-database-before-feature
+@virtual-product-file
+@reset-downloads-after-feature
+Feature: Add virtual product file from BO (Back Office).
+  As an employee I want to be able to add file of virtual product.
+
+  Scenario: I add virtual product file without limiting access days, download times and without expiration date
+    Given I add product "product1" with following information:
+      | name[en-US] | puffin icon |
+      | is_virtual  | true        |
+    And product "product1" should not have a file
+    And product product1 type should be virtual
+    When I add virtual product file "file1" to "product1" with following details:
+      | display name | puffin-logo.png |
+      | file name    | app_icon.png    |
+    Then product "product1" should have a virtual product file "file1" with following details:
+      | display name         | puffin-logo.png     |
+      | access days          | 0                   |
+      | download times limit | 0                   |
+      | expiration date      | 0000-00-00 00:00:00 |
```

Comment:
Same comment as the combination PR about empty DateTime values I think it's confusing to use `0000-00-00 00:00:00` as the value in tests when we expect the returned value to be `null`

I'd go for either use empty value here or use "null" string that would be converted into actual `null`